JASP-Engine/JASP/R/ttestbayesianpairedsamples.R

```diff
@@ -618,85 +618,90 @@ TTestBayesianPairedSamples <- function(dataset=NULL, options, perform="run", cal
 
 	results[["ttest"]] <- ttest
 
-	descriptives <- NULL
-
+	# descriptives <- list()
 	if (options$descriptives) {
-
-		descriptives <- list()
-
-		descriptives[["title"]] <- "Descriptives"
-
-		fields <- list(
-			list(name="v", type="string", title=""),
-			list(name="N",                  type="integer"),
-			list(name="mean", title="Mean", type="number", format="sf:4;dp:3"),
-			list(name="sd",   title="SD",   type="number", format="sf:4;dp:3"),
-			list(name="se",   title="SE",   type="number", format="sf:4;dp:3"))
-
-			## add credible interval values if asked for in plot
-			if (options$descriptivesPlots) {
-				interval <- 100 * options$descriptivesPlotsCredibleInterval
-				title <- paste0(interval, "% Credible Interval")
-				fields[[length(fields) + 1]] <- list(name = "lowerCI", type = "number",
-													 format = "sf:4;dp:3", title = "Lower",
-													 overTitle = title)
-				fields[[length(fields) + 1]] <- list(name = "upperCI", type = "number",
-													 format = "sf:4;dp:3", title = "Upper",
-													 overTitle = title)
-			}
-
-		descriptives[["schema"]] <- list(fields=fields)
-
-		descriptives.results <- list()
-
-		variables <- unlist(options$pairs)
-		variables <- unique(variables)
-		variables <- variables[variables != ""]
-
-		for (variable in variables) {
-
-			if (perform == "run") {
-
-				result <- try (silent = TRUE, expr = {
-
-					variableData <- dataset[[.v(variable)]]
-					variableDataOm <- na.omit(variableData)
-
-					posteriorSummary <- .posteriorSummaryGroupMean(variable=variableDataOm, descriptivesPlotsCredibleInterval=options$descriptivesPlotsCredibleInterval)
-					ciLower <- .clean(posteriorSummary$ciLower)
-					ciUpper <- .clean(posteriorSummary$ciUpper)
-
-					n <- .clean(as.numeric(length(variableDataOm)))
-					m <- .clean(as.numeric(mean(variableDataOm)))
-					std <- .clean(as.numeric(sd(variableDataOm)))
-					if(is.numeric(std)){
-						se <- .clean(as.numeric(std/sqrt(n)))}
-					else
-						se <- .clean(NaN)
-
-						list(v=variable, N=n, mean=m, sd=std, se=se, lowerCI=ciLower, upperCI=ciUpper)
-				})
-
-				if (isTryError(result)) {
-
-					result <- list(v=variable, N="", mean="", sd="", se="", lowerCI="", upperCI="")
-				}
-
-			} else {
-
-				result <- list(v=variable, N=".", mean=".", sd=".", se=".", lowerCI=".", upperCI=".")
-			}
-
-			descriptives.results[[length(descriptives.results)+1]] <- result
-		}
-
-		descriptives[["data"]] <- descriptives.results
-		descriptives[["status"]] <- "complete"
-
-
+	  tempList <- .makeDescriptivesTableTTestBayesianPaired(dataset, options)
+	  tempList[["status"]] <- "complete"
+	  results[["descriptives"]][["descriptivesTable"]] <- tempList
 	}
 
-	results[["descriptives"]][["descriptivesTable"]] <- descriptives
+	# if (options$descriptives) {
```

Comment:
Just delete these lines. We have GitHub in case we want to go back to a previous version of a script.